### DISCUSSION TITLE
Any way to prevent scrolling and zooming?

### DISCUSSION THREAD

**Discussion by ET3D-ET3D** [epezent/implot#251] [Q&A]
I want to display a graph and query it. It would be most natural if I could use the left mouse button to draw a query region, but even if that's not possible, disabling the ability to use the left and right buttons and scroll wheel would be very helpful. I want the graph to remain as is at all times. How can I do this?

**Comment by ET3D-ET3D** [comment]
Answering my one question: using "ImPlotFlags_NoBoxSelect, ImPlotAxisFlags_AutoFit, ImPlotAxisFlags_AutoFit" achieves this.

It would still be nice if querying could be done by a left click, but at least not having the others work helps.

**Reply by epezent** [reply]
Check out input mapping in `implot_internal.h`. It's a WIP and may not behave perfectly but it should allow you to switch LMB and RMB.

**Comment by SnowWindSaveYou** [comment]
same question
